### ISSUE TITLE
 No module named 'ros_g29_force_feedback'

### ISSUE THREAD

**Issue by EvilQQQ** [kuriatsu/ros-g29-force-feedback#25]
Hi !  
In other CARLA test files, my steering wheel has already encountered resistance. but when I was running   ~/catkin_ws/src/ros-g29-force-feedback/examples$  python3.8 carla_control.py， the error 'No module named' ros_g29_force_feedback 'was reported. May I ask how to resolve it? Or is there any other way to test whether the steering wheel can be controlled in reverse through code?

**Comment by kuriatsu** [comment]
Did you build this package?
```bash
cd catkin_ws
colcon build
```

**Comment by CaMi1le** [comment]
> Did you build this package?
> 
> ```shell
> cd catkin_ws
> colcon build
> ```

yes, I am trying ubuntu20.04 with ros2-galactic, cmake version 3.16. It still wont work when trying `ros2 run ros_g29_force_feedback g29_force_feedback --ros-args --params-file ros2_ws/src/ros_g29_force_feedback/config/g29.yaml `

**Comment by kuriatsu** [comment]
@CaMi1le You mean you had solved the issue?

**Comment by CaMi1le** [comment]
> @CaMi1le You mean you had solved the issue?

yes, and a simple advice, the readme may need some update to support the current repo

**Comment by kuriatsu** [comment]
@CaMi1le 
Okay, `no module ...` issue when `carla_control.py` seems to be caused by the conflict between conda and ros.
There may also be conflict among multiple ros versions (I'm not sure whether it causes the python error, though).
